library/pk_wrap.c

```diff
@@ -141,6 +141,82 @@ static int rsa_decrypt_wrap( void *ctx,
                 olen, input, output, osize ) );
 }
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+static int rsa_encrypt_wrap( void *ctx,
+                    const unsigned char *input, size_t ilen,
+                    unsigned char *output, size_t *olen, size_t osize,
+                    int (*f_rng)(void *, unsigned char *, size_t), void *p_rng )
+{
+    mbedtls_rsa_context * rsa = (mbedtls_rsa_context *) ctx;
+    int ret = MBEDTLS_ERR_ERROR_CORRUPTION_DETECTED;
+    psa_key_attributes_t attributes = PSA_KEY_ATTRIBUTES_INIT;
+    mbedtls_svc_key_id_t key_id = MBEDTLS_SVC_KEY_ID_INIT;
+    psa_status_t status;
+    mbedtls_pk_context key;
+    int key_len;
+    /* see RSA_PUR_DER_MAX_BYTES in pkwrite.c */
```

Comment:
Maybe it would be good idea moving `RSA_PUR_DER_MAX_BYTES` to the header file?

Reply:
Probably yes since similar RSA & ECDSA values are using in similar PRs

Reply:
I noticed while looking on decryption PR. `RSA_PUR_DER_MAX_BYTES` -> `RSA_PUB_DER_MAX_BYTES`.

Reply:
@mpg Should `RSA_PUR_DER_MAX_BYTES` go in `include/mbedtls/pk.h` or another header in `library/` ? ?

Reply:
I think `pk.h` - after all, it can be useful for users too. PSA has public macros for all kinds of sizes, and I think it's a good thing.

Reply:
Will be done in #5581 instead of currently pushed solution, will need to rebase when merged